pkgs/by-name/an/antimatter-dimensions/app.js

```diff
@@ -0,0 +1,14 @@
+const { app, BrowserWindow } = require('electron');
```

Comment:
Maybe it might make sense to use a more lightweight library such as https://www.npmjs.com/package/webview-nodejs?

Reply:
nixpkgs doesn't have these node packages.

Reply:
Fair enough. Feel free to mark this as resolved.